Dockerfile-alpine.template

```diff
@@ -3,6 +3,8 @@ FROM alpine:3.4
 ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 0.0.0
 
+COPY patches
```

Comment:
@SimenB I slept on your comment about the `COPY patches` last night. I came up with a more proper way for the patch. In my updated change, I always copy patches and remove the patches after applying the patches. And each patch should use the target architecture as prefix, i.e. `ppc64le-something.sh`. Inside the Dockerfile, it will use `/patches/$ARCH*` to select the corresponding patch files and apply patch.

Thanks.